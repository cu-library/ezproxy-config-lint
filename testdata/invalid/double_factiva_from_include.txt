IncludeFile ../valid/factiva.txt

Title Factivia Again 
URL https://global.factiva.com/en/sess/login.asp?xsid=[PLACE-XSID-HERE]
HJ www.factiva.com
