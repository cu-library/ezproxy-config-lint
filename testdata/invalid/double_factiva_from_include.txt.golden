testdata/invalid/double_factiva_from_include.txt:4: URL https://global.factiva.com/en/sess/login.asp?xsid=[PLACE-XSID-HERE] ← Origin already seen at "testdata/valid/factiva.txt:12" (L2002)
testdata/invalid/double_factiva_from_include.txt:5: HJ www.factiva.com ← Origin already seen at "testdata/valid/factiva.txt:13" (L2002)
